index.js

```diff
@@ -63,7 +63,7 @@ client.on('message', async (message) => {
 
   const announcementChannel = '953362393071775814'
 
-  if (message.content.toLocaleLowerCase() === '!announcement' && message.channel.id === announcementChannel){
+  if (message.content.toLowerCase() === '!announcement' && message.channel.id === announcementChannel){
```

Comment:
Added in that announcement channel variable and had the command be lower case. Figured this way there wouldn't be any issue if the Staff member did `!Announcement` or `!announcement`. Just to help avoid any confusion.